memory_view.c

```diff
@@ -108,6 +108,8 @@ static void
 unregister_exported_object(VALUE obj)
 {
     RB_VM_LOCK_ENTER();
+    if (!exported_object_table)
+        return;
```

Comment:
If we want to do this after `RB_VM_LOCK_ENTER()`, we should call `RUBY_VM_LOCK_LEAVE()` before `return`.

Reply:
OMG! Exactly!